src/internal/Subscription.ts

```diff
@@ -28,8 +27,6 @@ export class Subscription implements SubscriptionLike {
    */
   public closed = false;
 
-  private _parentage: Subscription[] | Subscription | null = null;
```

Comment:
Almost all of the code removed in this file (which is substantial) comes from no longer tracking parents on children, and instead just having the children remove themselves. Side effects of that are explained in other comments.